src/dataset.rs

```diff
@@ -26,10 +27,10 @@ pub struct Dataset {
     version: String,
 }
 
-pub fn build_dataset(root: std::path::PathBuf, day_0: DateTime<Utc>) -> Dataset {
-    let manifest_path = root.join("manifest.json");
+pub fn build_dataset(name: String, root: std::path::PathBuf, day_0: DateTime<Utc>) -> Dataset {
+    let dataset_path = root.clone().join(name.clone());
```

Comment:
`root` doesn't need to be cloned here, it doesn't get used anywhere else. This will take ownership of `manifest_root` in `init_dataset.rs` but I don't think we'll need to use that variable elsewhere.

Reply:
Good catch, fixed now